lib/ansible/plugins/filter/from_json.yml

```diff
@@ -4,7 +4,7 @@ DOCUMENTATION:
   short_description: Convert JSON string into variable structure
   description:
     - Converts a JSON string representation into an equivalent structured Ansible variable.
-    - Ansible internally auto-converts JSON strings into variable structures in most contexts, this plugin is used for those contexts it doesn't.
+    - Ansible internally auto-converts JSON strings into variable structures in most contexts, this plugin is used for those contexts it does not.
```

Comment:
```suggestion
    - Ansible automatically converts JSON strings into variable structures in most contexts, use this plugin in contexts where automatic conversion does not happen.
```
Any idea what contexts? An example might be very useful here.

Reply:
I sometimes use it in combination with the `file` lookup and when trying to access a field from the JSON in the same template evaluation. Not sure whether that's a really good example though.

Reply:
file lookup should be converted already